src/components/logs/logsWebview.ts

```diff
@@ -126,4 +140,16 @@ export class LogsPanel extends WebPanel {
             .replace('<!-- meta http-equiv="Content-Security-Policy" -->', meta);
     }
 
+    private getLogViewerOptions(): object {
+        const configuration = vscode.workspace.getConfiguration();
+        const follow = configuration.get('vscode-kubernetes.log-viewer.follow', false);
```

Comment:
Better to move these to `config.ts` for simmetry and so that config remains responsible for knowing the key names.